src/script/index.js

```
@@ -40,3 +40,13 @@ newsletterForm.addEventListener('submit', (e) => {
     const email = newsletterForm.email.value;
     alert(`You have successfully subscribed with the email : ${email}`);
 });
+const accordionBtn = document.getElementsByClassName('accordion-card__header');
+
+for (let i of accordionBtn) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'i' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)